### PR TITLE
Chnages in Question Module Backend

### PR DESCRIPTION
**Pull request by meetkumar-tatva**

### CODE REVIEW

File: APTITUDE/APTITUDETEST.Data/Data/QuestionRepository.cs [hunk 1]
```diff
@@ -731,6 +749,51 @@ private bool ValidateTopics(ImportQuestionFieldsVM question)
             return true;
         }
 
+        private string GenerateSequence(int duplicateId)
+        {
+            if (duplicateId == 0)
+            {
+                int highestNumber = getHighestSequence();
+                return (highestNumber + 1).ToString();
```
Comment (UrvishaNayee) [inline APTITUDE/APTITUDETEST.Data/Data/QuestionRepository.cs:757]: replace two lines with one line
return (getHighestSequence() + 1).ToString();

File: APTITUDE/APTITUDETEST.Data/Data/QuestionRepository.cs [hunk 2]
```diff
@@ -731,6 +749,51 @@ private bool ValidateTopics(ImportQuestionFieldsVM question)
...
+            questions.ForEach(q =>
+            {
+                q.sequence = start.ToString();
+                start++;
+            });
+        }
+
+        private int getHighestSequence()
+        {
+            return _context.Questions.Select(x => x.Sequence).ToList()
```
Comment (UrvishaNayee) [inline APTITUDE/APTITUDETEST.Data/Data/QuestionRepository.cs:790]: check here - ToList() really required ?

File: APTITUDE/APTITUDETEST.Data/Data/QuestionRepository.cs (hunk 1 above)
Comment (meetkumar-tatva) [reply APTITUDE/APTITUDETEST.Data/Data/QuestionRepository.cs]: Resolved

File: APTITUDE/APTITUDETEST.Data/Data/QuestionRepository.cs (hunk 2 above)
Comment (meetkumar-tatva) [reply APTITUDE/APTITUDETEST.Data/Data/QuestionRepository.cs:789]: List is required as we are getting list of sequence and then ordering it

**Review by UrvishaNayee — APPROVED** [review]

**Pull request merged**